[2023-05-23T10:29:46.633+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_taskflow_api_v02.get_age scheduled__2022-01-09T00:00:00+00:00 [queued]>
[2023-05-23T10:29:46.696+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_taskflow_api_v02.get_age scheduled__2022-01-09T00:00:00+00:00 [queued]>
[2023-05-23T10:29:46.703+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 6
[2023-05-23T10:29:47.443+0000] {taskinstance.py:1327} INFO - Executing <Task(_PythonDecoratedOperator): get_age> on 2022-01-09 00:00:00+00:00
[2023-05-23T10:29:47.512+0000] {standard_task_runner.py:57} INFO - Started process 715 to run task
[2023-05-23T10:29:47.658+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'dag_with_taskflow_api_v02', 'get_age', 'scheduled__2022-01-09T00:00:00+00:00', '--job-id', '225', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmp3ssmrx8b']
[2023-05-23T10:29:47.687+0000] {standard_task_runner.py:85} INFO - Job 225: Subtask get_age
[2023-05-23T10:29:48.407+0000] {task_command.py:410} INFO - Running <TaskInstance: dag_with_taskflow_api_v02.get_age scheduled__2022-01-09T00:00:00+00:00 [running]> on host 8a6612510f4c
[2023-05-23T10:29:49.537+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Sai' AIRFLOW_CTX_DAG_ID='dag_with_taskflow_api_v02' AIRFLOW_CTX_TASK_ID='get_age' AIRFLOW_CTX_EXECUTION_DATE='2022-01-09T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2022-01-09T00:00:00+00:00'
[2023-05-23T10:29:49.539+0000] {python.py:183} INFO - Done. Returned value was: 19
[2023-05-23T10:29:49.738+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=dag_with_taskflow_api_v02, task_id=get_age, execution_date=20220109T000000, start_date=20230523T102946, end_date=20230523T102949
[2023-05-23T10:29:50.069+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2023-05-23T10:29:50.299+0000] {local_task_job_runner.py:115} ERROR - Received SIGTERM. Terminating subprocesses
[2023-05-23T10:29:50.344+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 143
[2023-05-23T10:29:51.627+0000] {taskinstance.py:2651} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-05-23T10:29:51.707+0000] {taskinstance.py:2651} INFO - 0 downstream tasks scheduled from follow-on schedule check
